03_homework/homework.sh

```diff
@@ -7,21 +7,35 @@
 # On your terminal, input all the commands you have used to create the following:
 
 # 1. How would you create 5 directories? Feel free to use any name for your directories.
-
+mkdir dir1 dir2 dir3 dir4 dir5
 # 2. How would you verify the creation of all 5 directories?
-
+ls
 # 3. In each directory, how would you create 5 .txt files and write "I love data" into each within the directories?
+touch file1.txt file2.txt file3.txt file4.txt file5.txt dir1
+code file1.text
+I love data
+cp -r dir1 dir2/
+cp -r dir1 dir3/
+cp -r dir1 dir4/
+cp -r dir1 dir5/
 
 # 4. How would you verify the presence of all 5 files?
-
+ls -al
 # 5. How would you append to one of the existing files " and machine learning!"?
+code file1.txt
+add "and machine learning!"
 
 # 6. How would you verify that the text was indeed appended to the existing file?
+git status
+git add file1.txt
+git commit
```

Comment:
Please don't use git add/commit to test if the text is appended to the existing file.
Use `cat`